homeassistant/components/fronius/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
```

Comment:
```py

    FroniusSensorEntityDescription(
        key="meter_location",
        entity_category=EntityCategory.DIAGNOSTIC,
        value_fn=int,  # type: ignore[arg-type]
    ),
```

Reply:
Meter location 0, 1, 3 and 4 are known. These are properly translated in a different entity.
Location numbers > 255 are used for subload meters. I guess this number can be used to identify the meter.
May be a good candidate for `entity_registry_enabled_default=False,` as subload meters are probably not very common.

Reply:
But what does this mean for the user? Can we make these an enum sensor?

Reply:
there is 
```py
    FroniusSensorEntityDescription(
        key="meter_location_description",
        response_key="meter_location",
        entity_category=EntityCategory.DIAGNOSTIC,
        device_class=SensorDeviceClass.ENUM,
        options=[opt.value for opt in MeterLocationCodeOption],
        value_fn=get_meter_location_description,
    ),
```
providing an enum sensor. For "Subload" state, the numeric representation might still be useful.